package-lock.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "opensource-il-site",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
please don't commit this file.

all the changes in this file should be cancelled.